CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to re2c
+
+[re2c][re2c] is an open source project and a volunteer effort. re2c welcomes
+contribution from everyone. The following summarizes the process for
+contributing changes. Please take a moment to review this document in order to
+make the contribution process easy and effective for everyone involved.
+
+## Community
+
+re2c is maintained and supported by [Ulya Trofimovich][skvadrik] and developed
+in collaboration with a productive community of contributors. Please use the
+[GitHub issues][issues] to raise discussion of development topics.
+
+## Contributions
+
+Contributions to re2c should be made in the form of GitHub pull requests. Each
+pull request will be reviewed by current [project maintainer][skvadrik].
+Feedback can be provided and potentially changes requested or the pull request
+will be merged. All contributions should follow this format, even those from
+core contributors.
+
+## Language Standard
+
+The main project language is C++. re2c uses C++98 language standard at this time.
```

Comment:
I'd rather not stress C++98 too much, I think it will be good to move to C++11 in the near future, as it's becoming more of an obstacle than an advantage. On the other hand, I'd like to stress the need for code clarity and simplicity. Let's make it:
```
## Language guidelines

re2c is written in C++. The project uses a conservative language subset
(for portability reasons the currently used standard is C++98). The code
should stay simple and avoid complex object hierarchies and excessive
use of template metaprogramming. It should allow the reader to concentrate
on the underlying algorithms, without the language getting in the way.

The line length limit is 90 characters, indentation is 4 spaces, and the
indentation style is more or less K&R (egyptian braces).
```

Reply:
> I think it will be good to move to C++11 in the near future

Oh! This was on of my planned proposal in the near future :) Great news!